core/build.sbt

```diff
@@ -46,8 +46,6 @@ pomExtra := {
   </developers>
 }
 
-osgiSettings
```

Comment:
I misread the documentation for sbt-osgi, it seems that `osgiSettings` is still needed if we want to override the `publish` behavior. I reverted those changes at 131df70.